pkg/test/pods.go

```diff
@@ -33,7 +33,7 @@ type PodOptions struct {
 	InitImage                     string
 	NodeName                      string
 	PriorityClassName             string
-	InitResourceRequirements      v1.ResourceRequirements
+	InitContainers                []InitContainersOptions
```

Comment:
```suggestion
	InitContainers                []v1.Container
```

Could just use the `v1.Container` to simplify and give everyone more options